src/zcl/definition/cluster.ts

```diff
@@ -1761,7 +1761,7 @@ const Cluster: {
                     {name: 'numoftrans', type: DataType.uint8},
                     {name: 'dayofweek', type: DataType.uint8},
                     {name: 'mode', type: DataType.uint8},
-                    {name: 'thermoseqmode', type: BuffaloZclDataType.LIST_UINT16},
+                    {name: 'transitions', type: BuffaloZclDataType.LIST_THERMO_TRANSITIONS},
```

Comment:
I might be wrong but I think now we also need write transition. For LIST_UINT16 it was already implemented  but now it is new type...

Reply:
To be more precise, I think in setWeeklySchedule makes sense to use the same type and in this case write transformation will be required.